src/consumption.cpp

```diff
@@ -1,17 +1,19 @@
 #include "player.h"
-#include "monster.h"
+
+#include "addiction.h"
```

Comment:
The churn here is alphabetizing the includes, the only real change is adding options.h.